exampleSite/config.toml

```diff
@@ -1,8 +1,8 @@
 baseURL = "https://www.example.com"
 languageCode = "en-us"
 title = "My Digital Garden"
-theme = "hugo-digital-garden-theme"
-themesdir = "../../"
+theme = "digital-garden"
+themesdir = "./themes"
```

Comment:
The ./themes is not the right dir. It doesn't work.

Reply:
@nanqunchen It works for me here: https://github.com/IdiosApps/IdiosApps.github.io
When the sample config is moved into the root directory, it goes up two levels and thus `themes` is pointed to this `./themes`

Reply:
Let me explain why using `themesdir = "../../"` in the exampleSite directory may be a good choice.
If we just want to test/demo in `theme/exampleSite`, we can run the server command in the directory:
`> cd themes/hugo-digital-garden-theme`
`> hugo server`

You can copy/paste what you need to the project root dir, not the whole directory, and update `config.toml`.

That enables easy comparison between your site and the exampleSite while customizing, you can run the `hugo server` command in the root directory to start your site at the same time. The second server will automatically start at another port not default 1313.

---

However, I just checked the theme NexT's doc, it seems using the exampleSite as a Template is more popular. I agree that it is quite faster and more friendly to non-developers.

Reply:
That makes sense. I guess whichever way we go, a bit more documentation will be helpful.

Reply:
#5 please see the updated document commit.

Reply:
@nanqunchen Looks great, I'll close this PR.